npm/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
the convention seems to be to name your package folder `package` (instead of `npm`), could we do that?

Reply:
Package is slightly ambiguous since we also produce a nuget package, however that's probably fine here. I'll opt for convention here